M2/Macaulay2/packages/A1BrouwerDegrees.m2

```diff
@@ -32,7 +32,7 @@ newPackage(
 	 Email=>"gong@bowdoin.edu",
 	 HomePage=>"https://wgabrielong.github.io/"}
 	},
-    Headline=>"for working with A1-Brouwer degree computations",
+    Headline=>"A1-Brouwer degree computations",
```

Comment:
I'm trying to make all package headlines uniform, which includes removing redundant cluses. @tbrazel is this change okay with you?

Reply:
Looks good to me! 👍